### ISSUE TITLE
Support  `lotus g resource [resource_name]`

### ISSUE THREAD

**Issue by Nerian** [lotus/lotus#385]
Hi,

Lotus currently support generating actions. It may be useful to also support generating a complete resource.

Example:

Calling `lotus g resources books` would be equivalent to calling:

```
lotus g resources web books/index
lotus g resources web books/show
lotus g resources web books/new
lotus g resources web books/create --method=post
lotus g resources web books/edit
lotus g resources web books/update --method=patch
lotus g resources web books/destroy --method=delete

# routes.rb
resources :books
```

Calling `lotus g resource account` would be equivalent to calling:

```
lotus g resources web account/show
lotus g resources web account/new
lotus g resources web account/create --method=post
lotus g resources web account/edit 
lotus g resources web account/update --method=patch
lotus g resources web account/destroy --method=delete

# routes.rb
resource :account
```

So if we could support this new command, we could be saving some time since it makes the work of 6.

Thoughts?

**Comment by joneslee85** [comment]
I remember this has been brought up in the past but the answer is no from @jodosha because resources is more close to CRUD-based app and considered a bad practice. @jodosha would have more opinions to weigh on in this

**Comment by jodosha** [comment]
@Nerian Hello and thanks for the suggestion. Lotus encourages to write apps via BDD/TDD, where you fix one failing test case at the time.

Now, pure BDD where you write one step at the time the production code is overkilling. So, for this reason we have the "action generator", it adds routes/action/view/template and relative unit tests. This serves for the purpose of generate all the Lotus code that helps you to 1. pass one test 2. have the code infrastructure to implement one specific [use case](https://en.wikipedia.org/wiki/Use_case).

In other words, Lotus helps you to model software around a set of specific behaviours that your application exposes. CRUD approach (and so "resource generator") adds tons of behaviours modeling the application around data, rather than behavior/use cases.

This reminds me of Rails' Scaffold Generator, which is an anti-pattern for me. :-1: 

Getting back to your specific case, if you need this feature, you can implement it with a couple of Bash scripts.

**Comment by Nerian** [comment]
Sounds reasonable. Thanks!
